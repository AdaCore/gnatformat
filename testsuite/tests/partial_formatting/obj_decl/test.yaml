description: With an Ada file and with an object declaration selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 7",
    "-SC 2",
    "-EL 7",
    "-EC 10",
  ]
status_code: 1
control:
- [XFAIL, "True",
   "unexpected output: waiting for issue #eng/libadalang/langkit!1126"]