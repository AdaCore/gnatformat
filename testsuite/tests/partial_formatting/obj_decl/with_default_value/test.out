<CompilationUnit main.adb:1:1-11:10>
main.adb(<ObjectDecl ["Me"] main.adb:3:1-3:155>) - 3:1-3:155
^
   Me : constant Trace_Handle :=
     Create
       ("Hello",
       Unit_Name => FooBar_1,
       Default => FooBar_2,
       Stream => FooBar_3,
       Factory => FooBar_4,
       Finalize => FooBar_5);$

